README.md

```diff
@@ -1 +1,215 @@
-# React-Capstone-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Memes Generator</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+<details>
+  <summary>
+    <h1>📗 Table of Contents</h1>
+  </summary>
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+</details>
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Space Travelers' Hub <a name="about-project"></a>
+
+**Memes generator** Gets an array of popular memes that may be captioned with Imgflip API. The size of this array and the order of memes may change at any time. When this description was written, it returned 100 memes ordered by how many times they were captioned in the last 30 days. Additional properties other than those listed below may be added in the future without warning, so do not assume the JSON structure of each meme will never have new properties.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+<li><a href="https://www.w3.org/html/" target="_blank"><img align="center" src="https://uxwing.com/wp-content/themes/uxwing/download/brands-and-social-media/html-icon.svg" alt="html5" width="55" height="55"/></a></li>
+<li><a href="https://www.w3schools.com/css/" target="_blank"><img align="center" src="https://uxwing.com/wp-content/themes/uxwing/download/brands-and-social-media/css-icon.svg" alt="css3" width="55" height="55"/></a></li>
+<li><a href="https://reactjs.org/" target="_blank"><img align="center" src="https://uxwing.com/wp-content/themes/uxwing/download/brands-and-social-media/react-js-icon.svg" alt="react js" width="55" height="55"/></a></li>
+<li><a href="https://redux.js.org/" target="_blank"><img align="center" src="https://uxwing.com/wp-content/themes/uxwing/download/brands-and-social-media/redux-icon.svg" alt="react js" width="55" height="55"/></a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Fetch Memes data**
+- **Render Memes**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://react-capstone-project.onrender.com/)
+
+-[Demonstration of the project](https://www.loom.com/share/c9852699e94a4402b988c3de192776f1)
```

Comment:
- [ ] I can't watch the video, kindly check if there is something wrong and update it 
![image](https://github.com/haftamudesta/React-Capstone-Project/assets/62705947/4317d8ff-f02e-4176-946c-73c4f9ec9e41)

Reply:
Well resolved ✔️ 

![image](https://github.com/haftamudesta/React-Capstone-Project/assets/38088818/1ed798c9-cd16-4f2c-8ade-b3674e0a240b)